f8a_worker/data_normalizer.py

```diff
@@ -442,6 +442,52 @@ def _handle_gofedlib(self, data):
         transformed['dependencies'] = dependencies
         return transformed
 
+    def _handle_go_glide(self, data):
+        """Handle metadata from golang's glide.
+
+        https://glide.readthedocs.io/en/latest/glide.yaml
+        """
+        def _import2dependencies(import_list):
+            # transform
+            # [{"package": "github.com/Masterminds/vcs", "version": "^1.2.0"},
+            #  {"package": "github.com/codegangsta/cli", "version": "f89effe8"}]
+            # to
+            # ["github.com/Masterminds/vcs ^1.2.0",
+            #  "github.com/codegangsta/cli f89effe81c1ece9c5b0fda359ebd9cf65f169a51"]
+            dependencies = []
+            for dep in import_list:
+                _dep = "{name} {version}".format(name=dep['package'],
+                                                 version=dep.get('version', '')).strip()
+                # TODO: there might be also dep['subpackages'], do we want to include them too ?
```

Comment:
I think we want dep['subpackages'] as well.

Reply:
See latest commit.